__tests__/index.js

```diff
@@ -99,3 +100,27 @@ test('maxed-out configuration dry-run', () => {
     `--env API_KEY=hawk.xxxxXXXXxxXXxxxXXxXX.xxxXXxxxXXxxXXxxxXXX --network test_net stackhawk/hawkscantastic:best ` +
     `stackhawk.yml stackhawktest.yml`);
 });
+
+test('cli dry-run', () => {
+  buildInput({
+    dryRun: 'true',
+    apiKey: 'hawk.xxxxXXXXxxXXxxxXXxXX.xxxXXxxxXXxxXXxxxXXX',
+    version: '2.1.0'
+  });
+  const inputs = utilities.gatherInputs();
+  const cliCommand  = utilities.buildCLICommand(inputs);
+  expect(cliCommand)
+      .toEqual(`hawk --api-key=hawk.xxxxXXXXxxXXxxxXXxXX.xxxXXxxxXXxxXXxxxXXX scan stackhawk.yml`);
+});
+
+test('get latest version', async () => {
+   await getLatestVersion().then(function (data) {
+     expect(data).toEqual('2.1.1');
```

Comment:
Seems like this will need to get updated often as we release new versions. Is that OK?

Reply:
Good point!  I was more using this for local testing of that function I wrote.  It doesn't add much so I'll pull it out.